hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/mysql/MySQLMappingJdbcSqlConnectorTest.java

```diff
@@ -30,4 +35,29 @@ public static void beforeClass() {
         initialize(new MySQLDatabaseProvider());
     }
 
+    // Text is not a standard SQL type. This test specific to MySQL
+    @Test
+    public void createMappingWithTextColumnType() throws Exception {
```

Comment:
The test is duplicated.

It would be better to have it in the parent and have assume on the database type. 

Or rather run it for all DBs and when we come across one, which doesn't support `TEXT` type, we can specifically exclude it. That way we will also get informed we adding a support for a new database what is and what isn't supported.

Reply:
rev : 5750495eedfd37a0c15e484adff25c12604d0002